lib/tasks/recordings.rake

```diff
@@ -42,4 +42,22 @@ namespace :recordings do
       end
     end
   end
+
+  desc 'Associate any existing recordings with a tenant'
+  task :addToTenant, [:tenant_id] => :environment do |_t, args|
+    tenant_id = args[:tenant_id]
+    unless tenant_id
+      puts('No tenant ID was provided')
+      exit(1)
+    end
+
+    Recording.all.each do |rec|
+      next if rec.metadata.exists?(key: tenant_id)
+      begin
+        Metadatum.create!(recording_id: rec.id, key: 'tenant-id', value: tenant_id)
```

Comment:
Maybe we can print a success message? Either per recording or at the end just to inform the user that everything worked